### PR TITLE
[besu] Updated Charts with dependencies and hooks

### PR DESCRIPTION
**Pull request by sownak**
Release 1.0.0 Changes for Besu
- Nodekey and other secret generation moved from separate chart to Hooks.
- Tessera secret generation also moved from separate chart to Hooks.
- TLS cert Generation for use with proxy is now a dependent chart called `besu-tlscert-gen`.
- `bevel-scripts`, `bevel-storageclass` and `bevel-vault-mgmt` are dependent Charts for new Chart `besu-genesis`.
- `besu-tessera-node` is dependent chart for `besu-node` depending on privacy flags.
- Independent jobs will no longer use Flux/Gitops
- All deployments will start with a `genesis` chart which does the startup
- The first organization in the network.yaml must be a validator

Deprecations:
- `ethash` and `raft` consensus
- Add member/validators are not tested with current format. Will be released later.
- Generate crypto playbook also removed until the need for this playbook is established.
- `letsencrypt` certificates and `Prometheus` needs to be tested again.

### CODE REVIEW

File: .github/workflows/release_charts.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@ jobs:
       - name: Install additional Helm repos
         run: |
           helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm repo add bevel https://sownak.github.io/bevel
```
Comment (suvajit-sarkar) [inline .github/workflows/release_charts.yml:41]: Do we require this helm repo ?

File: .gitignore [hunk 2]
```diff
@@ -38,3 +38,6 @@
 **/.vscode/
 # custom config templates
 *_custom.tpl
+**/charts/*.tgz
+**/files/*.json
+requirements.lock
```
Comment (suvajit-sarkar) [inline .gitignore:43]: Kindly add new line

File: platforms/substrate/configuration/deploy-network.yaml [hunk 3]
```diff
@@ -55,7 +55,6 @@
       name: "{{ playbook_dir }}/../../shared/configuration/roles/setup/scripts"
     vars:
       namespace: "{{ org.name | lower }}-subs"
-      network_type: "{{ network.type | lower }}"
```
Comment (suvajit-sarkar) [inline platforms/substrate/configuration/deploy-network.yaml:58]: Saw this change in other files as well. Did we change anything related to network.type or was this not used ?

**Review by suvajit-sarkar — CHANGES_REQUESTED** [review]
LGTM. Just few files need new line addition and network schema needs to be updated.

File: platforms/substrate/configuration/deploy-network.yaml (hunk 3 above)
Comment (sownak) [reply platforms/substrate/configuration/deploy-network.yaml:58]: This is not used

File: .gitignore (hunk 2 above)
Comment (sownak) [reply .gitignore:43]: Yes added for others.

File: .github/workflows/release_charts.yml (hunk 1 above)
Comment (sownak) [reply .github/workflows/release_charts.yml:41]: Yes for now, will have to be updated to Hyperledger once all charts are done

**Review by suvajit-sarkar — APPROVED** [review]
LGTM

**Pull request merged**